packages/fluentui/react/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
+import { WithAsProp, withSafeTypeForAs, ComponentEventHandler, FluentComponentStaticProps, ProviderContextPrepared } from '../../types';
+import { Accessibility, cardBehavior, CardBehaviorProps } from '@fluentui/accessibility';
+import { UIComponentProps, ChildrenComponentProps, commonPropTypes, createShorthandFactory } from '../../utils';
+import { useTelemetry, useStyles, getElementType, getUnhandledProps, useAccessibility } from '@fluentui/react-bindings';
+import * as PropTypes from 'prop-types';
+import * as _ from 'lodash';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import CardHeader from './CardHeader';
+import CardBody from './CardBody';
+import CardPreview from './CardPreview';
+import CardFooter from './CardFooter';
+import CardTopControls from './CardTopControls';
+
+export interface CardProps extends UIComponentProps, ChildrenComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * */
+  accessibility?: Accessibility<CardBehaviorProps>;
+
+  /**
+   * Called on click.
+   *
+   * @param event - React's original SyntheticEvent.
+   * @param data - All props.
+   */
+  onClick?: ComponentEventHandler<CardProps>;
+
+  /**
+   * Do not show padding for the card root
+   */
+
+  noPadding?: boolean;
+
+  /**
+   * Use card horizontal layout
+   */
+
+  horizontal?: boolean;
+
+  /**
+   * Center content
+   */
+
+  center?: boolean;
```

Comment:
```suggestion
  centered?: boolean;
```

Or even `align`/`textAlign`

Reply:
I'd prefer to have `centered`, not `align/textAlign`